server/server.go

```diff
@@ -184,10 +184,14 @@ func (srv *Server) startServer(_ context.Context) error {
 
 	api := http.NewAPI(procedureManager, procedureFactory, manager)
 	httpService := http.NewHTTPService(srv.cfg.HTTPPort, time.Second*10, time.Second*10, api.NewAPIRouter())
-	err = httpService.Start()
-	if err != nil {
-		return errors.WithMessage(err, "start http service failed")
-	}
+	go func() {
+		err := httpService.Start()
```

Comment:
Any control method to close the http service?

Reply:
`http.Stop()` has been definition but not used. At present, the service does not need to actively shut down the http server.